### PR TITLE
Fix small typo by removing semicolon

### PR DESCRIPTION
**Pull request by mercuryseries**
Fix small typo by removing semicolon

### CODE REVIEW

File: security/experimental_authenticators.rst [hunk 1]
```diff
@@ -493,7 +493,7 @@ authenticator, you would initialize the passport like this::
             return new Passport($user, new PasswordCredentials($password), [
                 // $this->userRepository must implement PasswordUpgraderInterface
                 new PasswordUpgradeBadge($password, $this->userRepository),
-                new CsrfTokenBadge('login', $csrfToken);
```
Comment (OskarStark) [inline security/experimental_authenticators.rst:496]: please change this to a comma instead, thanks!

Comment (mercuryseries) [reply security/experimental_authenticators.rst:496]: Done ✅

**Review by mercuryseries — COMMENTED** [review]
Done ✅

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Honoré.

**Pull request merged**